templates/ts/sso-tab-with-obo-flow/api/README.md

```diff
@@ -18,17 +18,21 @@ You can add your logic to the single Azure Function created by this template, as
 
 ### Call the Function
 
-To call your Azure Function, the client sends an HTTP request with an SSO token in the `Authorization` header. The token can be retrieved using the TeamsFx SDK from your app's client (custom tab). Here is an example:
+To call your Azure Function, the client sends an HTTP request with an SSO token in the `Authorization` header. Here is an example:
 
 ```ts
-import { TeamsFx } from "@microsoft/teamsfx";
-
-const teamsfx = new TeamsFx();
-const accessToken = await teamsfx.getCredential().getToken("");
-// note: empty string argument on the previous line is required for now, this will be fixed in a later release
-const response = await fetch(`${functionEndpoint}/api/${functionName}`, {
+import { TeamsUserCredentialAuthConfig, TeamsUserCredential } from "@microsoft/teamsfx";
+
+const authConfig: TeamsUserCredentialAuthConfig = {
+  clientId: process.env.REACT_APP_CLIENT_ID,
+  initiateLoginEndpoint: process.env.REACT_APP_START_LOGIN_PAGE_URL,
+};
+const teamsUserCredential = new TeamsUserCredential(authConfig);
+const accessToken = await teamsUserCredential.getToken(""); // Get SSO token
+const endpoint = "https://YOUR_API_ENDPOINT";
+const response = await axios.default.get(endpoint + "/api/" + functionName, {
   headers: {
-    Authorization: `Bearer ${accessToken.token}`,
+    authorization: "Bearer " + accessToken.token,
```

Comment:
Why use lower case for `Authorization`?

Reply:
This should be updated by `prettier`, and I just reverted it back. Thanks!